multisig/proposals/arbitrumsepolia/vip-014/index.ts

```diff
@@ -0,0 +1,142 @@
+      target: PRIME,
+      signature: "transferOwnership(address)",
+      params: [arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: PLP,
+      signature: "transferOwnership(address)",
+      params: [arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    ...REWARD_DISTRIBUTORS.map(rewardDistributor => {
+      return {
+        target: rewardDistributor,
+        signature: "transferOwnership(address)",
+        params: [arbitrumsepolia.NORMAL_TIMELOCK],
+      };
+    }),
+    {
+      target: PSR,
+      signature: "transferOwnership(address)",
+      params: [arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: COMPTROLLER_BEACON,
+      signature: "transferOwnership(address)",
+      params: [arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: VTOKEN_BEACON,
+      signature: "transferOwnership(address)",
+      params: [arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: POOL_REGISTRY,
+      signature: "transferOwnership(address)",
+      params: [arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    ...COMPTROLLERS.map(comptroller => {
+      return {
+        target: comptroller,
+        signature: "transferOwnership(address)",
+        params: [arbitrumsepolia.NORMAL_TIMELOCK],
+      };
+    }),
+    ...VTOKENS.map(comptroller => {
+      return {
+        target: comptroller,
+        signature: "transferOwnership(address)",
+        params: [arbitrumsepolia.NORMAL_TIMELOCK],
+      };
+    }),
+    {
+      target: arbitrumsepolia.XVS_VAULT_PROXY,
+      signature: "_setPendingAdmin(address)",
+      params: [arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: XVS_STORE,
+      signature: "setPendingAdmin(address)",
+      params: [arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: XVS_BRIDGE_ADMIN_PROXY,
+      signature: "transferOwnership(address)",
+      params: [arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: XVS,
+      signature: "transferOwnership(address)",
+      params: [arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: arbitrumsepolia.RESILIENT_ORACLE,
+      signature: "transferOwnership(address)",
+      params: [arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: arbitrumsepolia.CHAINLINK_ORACLE,
+      signature: "transferOwnership(address)",
+      params: [arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: arbitrumsepolia.REDSTONE_ORACLE,
+      signature: "transferOwnership(address)",
+      params: [arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: BOUND_VALIDATOR,
+      signature: "transferOwnership(address)",
+      params: [arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: arbitrumsepolia.VTREASURY,
+      signature: "transferOwnership(address)",
+      params: [arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "grantRole(bytes32,address)",
+      params: [DEFAULT_ADMIN_ROLE, arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+  ]);
```

Comment:
We don't need this command on all the networks, its already done here
https://github.com/VenusProtocol/vips/blob/main/multisig/proposals/arbitrumsepolia/vip-009/index.ts#L114